tests/bluetooth/ctrl_isoal/src/main.c

```diff
@@ -20,1517 +20,7747 @@
 /* Include the DUT */
 #include "ll_sw/isoal.c"
 
-/*
- * Note: sdu_interval unit is uS, iso_interval is a multiple of 1.25mS
- *
- * pdus_per_sdu = burst_number * (sdu_interval / (iso_interval * 1250));
- *
- */
+/* #define DEBUG_TEST			(1) */
+/* #define DEBUG_TRACE			(1) */
 
-/* Global vars to be used at setup */
-isoal_sink_handle_t sink_hdl;
-isoal_status_t err;
-uint16_t handle;
-uint32_t stream_sync_delay = 1;
-uint32_t group_sync_delay = 1;
-uint16_t iso_interval = 1;
-uint32_t sdu_interval = (3 * 1250);
-uint8_t  burst_number = 1;
-uint8_t  flush_timeout = 1;
-uint8_t  role = BT_CONN_ROLE_PERIPHERAL;
-uint8_t  framed;
 
-uint64_t payload_number = 2000;
-
-/* This should point at start of reference PDU content when emit is called */
-uint8_t *pdu_ref;
-
-/* Sink as referenced by sink_hdl */
-struct isoal_sink *sink;
-
-/*
- * PDU defines and helper functions
- */
-#define TESTDATA_MAX_LEN 60
-uint8_t testdata[TESTDATA_MAX_LEN] = {
-	0xBF, 0x9B, 0xD1, 0x7D, 0x9E, 0xE0, 0xB9, 0xA4, 0x71, 0xE6,
-	0x80, 0xA7, 0x59, 0xAD, 0xB0, 0x98, 0xB3, 0x95, 0x83, 0x6B,
-	0xF8, 0xFE, 0xCB, 0xA1, 0xE9, 0x7A, 0xDD, 0x86, 0x68, 0x50,
-	0x77, 0x6E, 0xF2, 0x5C, 0xD7, 0x53, 0x62, 0x56, 0x74, 0x89,
-	0xCE, 0xC5, 0xAA, 0x65, 0x8C, 0xFF, 0xF5, 0x8F, 0xDA, 0xBC,
-	0xB6, 0xEC, 0x5F, 0xE3, 0xD4, 0xEF, 0xC8, 0x92, 0xC2, 0xFB
-};
+#define TEST_RX_PDU_PAYLAOD_MAX	(40)
+#define TEST_RX_PDU_SIZE		(TEST_RX_PDU_PAYLAOD_MAX + 2)
 
-uint8_t pdu_data[(TESTDATA_MAX_LEN+2)];
-struct node_rx_iso_meta meta;
-struct isoal_pdu_rx pdu_meta = {&meta, (struct pdu_iso *) &pdu_data};
+#define TEST_RX_SDU_FRAG_PAYLOAD_MAX	(100)
 
 #define LLID_TO_STR(llid) (llid == PDU_BIS_LLID_COMPLETE_END ? "COMPLETE_END" : \
 	(llid == PDU_BIS_LLID_START_CONTINUE ? "START_CONT" : \
 		(llid == PDU_BIS_LLID_FRAMED ? "FRAMED" : \
 			(llid == PDU_BIS_LLID_CTRL ? "CTRL" : "?????"))))
 
+#define DU_ERR_TO_STR(err) (err == 1 ? "Bit Errors" : \
```

Comment:
I think it is bad practice to use a common variable name such as `err` as a macro argument: Suggest to either prefix it with `_` (as `_err`) or to use a less common name (maybe `du_err`?)